test/test_color.py

```diff
@@ -75,6 +81,7 @@ def test_retry_on_connection_error():
         assert len(history) == 3
 
 
+@pytest.mark.geoportail
```

Comment:
This test does not connect to geoportail. It was used manually to test time, delay and  factor params.

You can remove the marker `@pytest.mark.geoportail`